src/app/application.cpp

```diff
@@ -278,7 +279,7 @@ void Application::torrentFinished(BitTorrent::TorrentHandle *const torrent)
         program.replace("%T", torrent->currentTracker());
         program.replace("%I", torrent->hash());
 
-        QProcess::startDetached(program);
+        std::thread ([program]() { return system(program.toStdString().c_str()); }).detach();
```

Comment:
Actually I'm thinking I should add a note in the UI:
```
It is recommended to put commands to a script file and invoke it here.
```

>instead of applying this useless constraint.

Well if it works for most users then it's not a bad idea.